pom.xml

```diff
@@ -216,6 +216,23 @@
 				<artifactId>appengine-maven-plugin</artifactId>
 				<version>${app-engine-maven-plugin.version}</version>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-javadoc-plugin</artifactId>
+				<configuration>
+					<excludePackageNames>
+						com.example,
+						com.example.*,
+						<!-- Exclude samples and classes in the Firestore Google Client library packages -->
+						com.google.cloud.firestore
+					</excludePackageNames>
+					<links>
+						<link>https://googleapis.dev/java/google-cloud-vision/latest/</link>
```

Comment:
What's the purpose of these links?

Reply:
It makes it so that any client library javadoc classes referenced in the spring-cloud-gcp javadoc will link successfully to the client library javadocs.

Reply:
Cool! How do we know that we covered all of such references?

Reply:
It's a best effort guess :(